### DISCUSSION TITLE
饭

### DISCUSSION THREAD

**Discussion by cpu644** [share121/inter-knot#2378] [其他]
![IMG_6489](https://github.com/user-attachments/assets/0380a654-959d-4a89-88a1-0b89d878a9b9)
![IMG_6498](https://github.com/user-attachments/assets/148d5ff0-2db1-414e-990d-4fb979eafaa9)

**Comment by Genshin-Neuvillette** [comment]
豪赤！

**Comment by YuTinMIn** [comment]
吃剩的骨头不要扔，留着做排骨泡饭
